study.md

```diff
@@ -44,4 +51,10 @@ instead of `hash` for `location`?
 
 ```md
 <!-- your answer here -->
+URLs link one piece of data to another. The hash option uses the URL's anchor to load the initial
+state of your app.  It will keep your site in sync as you move from view to view.  Your app uses the browser's history
+API to procduce URLs with the structure /something/something-else.  If history is not supported by the browser, the app
+will fall back to hash.  When deploying to git-hub pages you must be aware of the build process.  You must make sure you
+run ember build --enviroment production and build the project to dist/.
```

Comment:
Yes, and also to use `hash` mode. GHPages doesn't run an Ember server, so it can't just serve up new pages in response to URLs; it can only serve up existing, hard-coded pages. Ember can get around this by running as JS from the root page, and parsing the part of the URL after `#` to figure out which Route to load.